src/backend/api/v1/Users.ts

```diff
@@ -120,4 +122,14 @@ usersRouter.delete('/deleteAvatar', (req, res, next) => {
     fs.unlink('./public' + req.body.oldPath.substring(1, req.body.oldPath.length), (err) => {
         res.send('Deleted')
     });
-});
\ No newline at end of file
+});
+
+//recommend user algorithm start
+usersRouter.get('/recommend/:_id', async (req, res, next) => {
+    const userId = req.params._id;
+    const currentUserSchema = await User.find({_id: userId}).exec();
+    const currentUser = currentUserSchema[0];
+    const userList = await User.find({}).exec(); 
```

Comment:
I think userList would include `currentUser` in it, which might have the highest score because they are the same person. Maybe we want to avoid that?

Reply:
I will exclude currentUser in the userList